### ISSUE TITLE
Exception on cache:clear --env=prod using doctrine/common 2.8

### ISSUE THREAD

**Issue by sweoggy** [Sylius/Sylius#8416]
As title, I get the following exception:

```
[Doctrine\Common\Proxy\Exception\InvalidArgumentException]                   
Unable to create a proxy for a final class "Sylius\Component\Addressing\Model\AddressLogEntry".
```

When updating doctrine/common to 2.8, 2.7.3 works fine. Exception does not occur in dev environment.

**Comment by harmstyler** [comment]
There appear to be a number of Sylius models that are marked Final. According to Doctrine, a model must not be Final. As of Doctrine 2.8 they have started throwing exceptions for this. All of the models need to be made extensible, but in the meantime I'd recommend Sylius move the doctrine/common requirement to be `2.7.*`

**Comment by pjedrzejewski** [comment]
I believe this is fixed in #8385.
